endaq/ide/info.py

```diff
@@ -16,22 +16,34 @@
 from .measurement import ANY, get_channels
 
 
+__all__ = [
+    "parse_time",
+    "format_channel_id",
+    "format_timedelta",
+    "format_timestamp",
```

Comment:
These are primarily for internal use and shouldn't be in `__all__`. If, in an edge-case, a user needs them, they can be imported explicitly.

Reply:
can you tell me which ones exactly are intended for "internal use"? I can't tell the difference. (related note: should the "internal use" functions have names prefixed with underscores?)

Reply:
> can you tell me which ones exactly are intended for "internal use"? I can't tell the difference.

The four highlighted. 

> (related note: should the "internal use" functions have names prefixed with underscores?)

By convention, the underscore implies a function shouldn't be used. These aren't *exclusively* for internal use; they might be useful to some, but most users won't care. I'd prefer they were excluded from `__all__`, but honestly, it really doesn't matter too much.

Reply:
it makes no difference to me so I don't mind. addressed in 3772262ad285ab6ed6cd9b45c5c3bba7ffdbaca0